packages/cli/src/WorkflowHelpers.ts

```diff
@@ -373,15 +374,21 @@ export async function replaceInvalidCredentials(workflow: WorkflowEntity): Promi
  * Returns all IDs if user is global owner (see `whereClause`)
  */
 export async function getSharedWorkflowIds(user: User, roles?: string[]): Promise<string[]> {
```

Comment:
can you also please change the signature here 
```suggestion
export async function getSharedWorkflowIds(user: User, roles?: RoleNames[]): Promise<string[]> {
```